lib/crypto/ed25519/ed25519.go

```diff
@@ -44,6 +44,20 @@ type PublicKey ed25519.PublicKey
 // PublicKeyBytes is an encoded ed25519 public key
 type PublicKeyBytes [PublicKeyLength]byte
 
+// SignVerify verify signature given pubkey and msg
```

Comment:
nit fixing up the comment :wink: 

```suggestion
// SignVerify verifies a signature given a public key and a message
```